photon-core/src/main/java/org/photonvision/vision/calibration/CameraCalibrationCoefficients.java

```diff
@@ -201,4 +203,19 @@ public String toString() {
                 + Arrays.toString(distCoeffsArr)
                 + "]";
     }
+
+
+    public CameraCalibrationCoefficients cloneWithoutObservations() {
+        // yeet observations, keep all else
+        var newobs = new ArrayList<>(observations);
+        var remapped = new ArrayList<BoardObservation>();
+        for (var o : observations) {
+            var r = o.clone();
+            r.snapshotData = null;
+            remapped.add(r);
+        }
+
+        return new CameraCalibrationCoefficients(resolution, cameraIntrinsics, distCoeffs, calobjectWarp, remapped, calobjectSize, calobjectSpacing, lensmodel);
+    }
```

Comment:
This is the magic sauce to remove snapshots. This reduces fullsettings to 463kb; deleting observations entirely gets us down to 5218 bytes for a 30 snapshot calibration